ting_material_details.field.inc

```diff
@@ -457,6 +463,36 @@ function ting_material_details_field_formatter_view($entity_type, $entity, $fiel
             '#markup' => $entity->serieDescription,
           );
           break;
+
+        case 'ting_details_is_part_of_search':
+          $type = $uri = NULL;
+          foreach ($entity->getRelations() as $relation) {
+            $type = explode(':', $relation->getType());
+            if ( isset($type[1]) && $type[1] == 'isPartOfManifestation' ) {
+              $uri  = $relation->getURI();
+            }
+          }
+          $render['#prefix'] = '<span class="ting-material-detail ting-material-is-part-of">';
+          $render['#suffix'] = '</span>';
+          if (!is_array($val)) {
+            $val = array($val);
+          }
+          if ( $uri ) {
+            foreach ($val as $key => $detail) {
+              $render['part_of'][$key]['#theme'] = 'link';
+              $render['part_of'][$key]['#text'] = $detail;
+              $render['part_of'][$key]['#path'] = 'search/ting/' . $uri;
```

Comment:
I do not understand this part of the code.

Line 472 extracts a single URI from the relations. This loop creates a link for each value of the field. If there are multiple values for the field this will result in multiple links all linking to the same search.

Is this intentional?